test/jest.setup.js

```diff
@@ -28,4 +28,4 @@ afterAll(async () => {
   }
 })
 
-jest.setTimeout(process.env.JEST_TIMEOUT || 30000)
+jest.setTimeout(process.env.JEST_TIMEOUT || 60000)
```

Comment:
please rollback to default timeout, if you are having issues with specific test please change it in the specifit test